### ISSUE TITLE
Can't uninstall on Windows 10

### ISSUE THREAD

**Issue by maopequena** [localsend/localsend#2048]
### Describe the bug

I've had a notification for un update for a while now. I use UniGetUI to update my apps. Today I decided to update everything, and LocalSend won't update, won't uninstall and I can't find it's folder in my system. But it still appears as "to update" on both UniGetUI and Settings > Windows Applications (although it doesn't appear in Control Panel > Uninstall or Modify a Program). I've looked in my usual installation folders in all my drives and nothing shows. Already restarted my computer and cleared the cache and temp folders. It is still showing as an update, but now the app doesn't open. I don't understand...
I've also tried installing the new version, but it requests I uninstall the previous version... which I can't do.

### To reproduce

N/A

### Expected behavior

When requesting an update, from either UniGetUI or Settings > Apps, it should be updated. 
When requesting to uninstall, from either UniGetUI or Settings > Apps, it should be uninstalled. 

### Screenshots

![Captura de Tela (103)](https://github.com/user-attachments/assets/e1f3a461-5283-41d1-8147-d1aecd015f9c)
![Captura de Tela (104)](https://github.com/user-attachments/assets/f3f55492-6d40-40e6-9d75-e3293fffa883)
![Captura de Tela (106)](https://github.com/user-attachments/assets/17f3b6e8-d091-461a-b77e-399a6e857ccc)
![Captura de Tela (107)](https://github.com/user-attachments/assets/feb6e16f-f4e8-493a-8dae-47dc425f8f67)

### Desktop (please complete the following information)

![Captura de Tela (108)](https://github.com/user-attachments/assets/8bb5f6b4-117b-405a-b5a6-3e77424427e1)

**Comment by Tienisto** [comment]
On v1.14.0 and earlier, LocalSend provided exe and msix installers. You probably used the msix installer. Now, only the exe installer is provided that might require you to uninstall the old msix version first.

**Comment by maopequena** [comment]
Yeah, I had to reinstall the app so it would work. Now everything works fine. Thank you.
